src/core/geometry/qgstriangulatedsurface.h

```diff
@@ -0,0 +1,213 @@
+/***************************************************************************
+                         qgstriangulatedsurface.h
+                         -------------------
+    begin                : August 2024
+    copyright            : (C) 2024 by Jean Felder
+    email                : jean dot felder at oslandia dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSTRIANGULATEDSURFACE_H
+#define QGSTRIANGULATEDSURFACE_H
+
+#include "qgis_core.h"
+#include "qgis_sip.h"
+#include "qgspolyhedralsurface.h"
+
+class QgsTriangle;
+
+
+/**
+ * \ingroup core
+ * \class QgsTriangulatedSurface
+ * \brief triangulated surface geometry type
```

Comment:
Same here, can you expand these docs to give a lot more detail about this geometry type?

Reply:
Done.